packages/wrangler/src/dev.tsx

```diff
@@ -557,18 +557,6 @@ export async function startDev(args: StartDevOptions) {
 			logger.loggerLevel = args.logLevel;
 		}
 
-		if (args.local) {
-			logger.warn(
-				"--local is no longer required and will be removed in a future version.\n`wrangler dev` now uses the local Cloudflare Workers runtime by default. 🎉"
-			);
-		}
-
-		if (args.experimentalLocal) {
-			logger.warn(
-				"--experimental-local is no longer required and will be removed in a future version.\n`wrangler dev` now uses the local Cloudflare Workers runtime by default. 🎉"
-			);
-		}
```

Comment:
I think we want to maintain this warning, since we probably will be removing `--experimental-local`

Reply:
aight, brought it back